pkgs/tools/security/1password-gui/default.nix

```diff
@@ -2,10 +2,15 @@
 , fetchurl
 , appimageTools
 , makeWrapper
-, electron
+, electron_9
```

Comment:
I have a (weak) objection to this. This makes it harder to override packages. It is easier for a user to use

```
_1password-gui.override { electron = myOwnElectronFork }
```

Than

```
_1password-gui.override { electron_9 = myOwnElectronFork }
```

Also, it isn't stable, because the arguments of the derivation change with each major electron version update.

Reply:
I would argue that it's important to keep the major version of Electron the same when you override the package. Doing things "my way" make it more obvious that you are doing something you shouldn't.

Another reason is that we would need to use overrides for almost all packages requiring Electron which polutes `all-packages.nix`.

Lastly, I did the similar change treewide in 4d6411288c29adc48f16851fd83524c0944073d8 and I prefer to keep things consistent.

Reply:
As I said, only a weak objection. I agree that it is probably different than e.g. the JDK where usually any newer version that supports the bytecode level works.